src/platform/posix/posix_resolv_gai.c

```diff
@@ -264,6 +264,7 @@ resolv_ip(const char *host, const char *serv, int passive, int family,
 	    sizeof(item->name_buf)) {
 		NNI_FREE_STRUCT(item);
 		nni_aio_finish_error(aio, NNG_EADDRINVAL);
+		return;
 	} else {
```

Comment:
Generally speaking I prefer to avoid the else if you're returning.  I'll follow up with that later.

Reply:
Actually, maybe it would be best if you did that... as there is also another style nit.

Reply:
Thank you for your contribution.

There are two minor style nits that ideally I'd like you to resolve before I merge.  Best would be to resubmit (squashed and rebased) for a clean git history.

If you would rather not do that, then I can do it for you.  Please let me know within a day or so, because if I've not heard back then I'll probably just do it myself -- we definitely do want these changes.

Reply:
> Generally speaking I prefer to avoid the else if you're returning. I'll follow up with that later.

I hope I understand you correctly.